lib/whitespace.coffee

```diff
@@ -38,9 +43,18 @@ class Whitespace
       bufferDestroyedSubscription.dispose()
       bufferSavedSubscription.dispose()
 
-    @subscriptions.add(bufferSavedSubscription)
-    @subscriptions.add(editorDestroyedSubscription)
-    @subscriptions.add(bufferDestroyedSubscription)
+    @editorSubscriptions.add(bufferSavedSubscription)
+    @editorSubscriptions.add(editorDestroyedSubscription)
+    @editorSubscriptions.add(bufferDestroyedSubscription)
+
+  saveWithoutModifying: (editor) ->
```

Comment:
Unregister all the editor events, issue the save, and then re-register the editor events. This feels a bit more like a sledgehammer than a scalpel, but it isolates the changes so that they only have an effect when we use the new command.